pom.xml

```diff
@@ -2043,6 +2043,9 @@
         <weld3.version>3.0.0.Final</weld3.version>
         <xerces.version>2.11.0</xerces.version>
         <yasson.version>1.0.3</yasson.version>
+        <helidon.version>1.0.2</helidon.version>
```

Comment:
CQ required for Helidon, MP Config API, smallrye

Reply:
[CQ#: 19752](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19752)
[CQ#: 19756](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19756)